src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -234,14 +233,13 @@ export default function NurseryWithdrawalsDetails({
             <NurseryTransferContent />
           </Box>
         )}
-        {withdrawal?.purpose === 'Dead' && (
+        {(withdrawal?.purpose === 'Dead' || withdrawal?.purpose === 'Other') && (
```

Comment:
just looked at the Figma again, and it doesn't appear that there are differences between these and the Nursery Transfer details either. Would it make sense to include that here as well?

Reply:
good point, will do that

Reply:
Thanks, Karthik